### ISSUE TITLE
Error text is not accessible

### ISSUE THREAD

**Issue by piofinn** [fremtind/jokul#372]
The current implementation of error text is **only** distingushed as an error by the color. It should have some other sort of marker that distinguishes it as an error, so that it is easily separated from help text after failed validation. Maybe just a dot, like below?

![Skjermbilde 2019-09-23 kl  10 49 42](https://user-images.githubusercontent.com/25739615/65412608-df03c800-ddef-11e9-9d14-beab2883e891.png)

Let the discussion commence 🏁

**Comment by frisol97** [comment]
Is this still relevant?

**Comment by lfbergee** [comment]
Hvis vi skal begrense ikonbruk er kanskje dit eller kryss bra nok. Menja, er nok relevant enda

**Comment by Mikkelinski** [comment]
Tror dette bør sees på sammen med mulig ikonbehov for meldingsboksene våre hvor vi har samme problem.
